etc/utils.py

```diff
@@ -21,18 +21,18 @@ def get_random_word(words):
     if not words:
         raise Exception("Words list is empty!", words)
 
-    rand_idx = random.randint(0, len(words)-1)
+    rand_idx = random.choice(range(len(words)))
```

Comment:
You didnt understand `random.choice`.
`return random.choice(words)`
next time play with the function in IPython